recipes-omnect/initrdscripts/omnect-os-initramfs/factory-reset

```diff
@@ -102,6 +102,14 @@ factory_reset_wipe_custom() {
     return 0
 }
 
+get_factory_reset_status() {
+    echo -n "$(get_bootloader_env_var "${FACTORY_RESET_STATUS_VAR}")"
+}
+
+clear_factory_reset_status() {
```

Comment:
get these one-line functions called in multiple occurrences?

Reply:
no, right now only in omnect-device-service-setup.

Reply:
but i like to reuse the variable from the factory-reset modul

Reply:
> but i like to reuse the variable from the factory-reset modul

please elaborate. what does that to do with the usage of a one-line function?

Reply:
changed: removed function and used factory-reset variable direct in omnect-device-service-setup